packages/office-ui-fabric-react/src/components/DetailsList/__snapshots__/DetailsList.test.tsx.snap

```diff
@@ -5363,6 +5363,7 @@ exports[`DetailsList renders List with hidden checkboxes correctly 1`] = `
                                           {
                                             padding-right: 8px;
                                           }
+
```

Comment:
This is weird. I wonder what's causing this.

Reply:
Could be IDE

Reply:
?? It's a snapshot file, I'm not sure how IDE matters

Reply:
If you have a plugin or somesuch and manually save the snapshot file prior to commit this can happen. Otherwise yea idk.